pkg/server/sandbox_run.go

```diff
@@ -180,10 +180,24 @@ func (c *criContainerdService) RunPodSandbox(ctx context.Context, r *runtime.Run
 			id, err)
 	}
 
-	// Add sandbox into sandbox store.
-	meta.CreatedAt = time.Now().UnixNano()
+	// Setup network.
 	// TODO(random-liu): [P2] Replace with permanent network namespace.
 	meta.NetNS = getNetworkNamespace(createResp.Pid)
+	if err = c.netPlugin.SetUpPod(meta.NetNS, config.GetMetadata().GetNamespace(), name, id); err != nil {
+		return nil, fmt.Errorf("failed to setup network %v", err)
+	}
+	defer func() {
+		if retErr != nil {
+			// Teardown network if an error is returned.
+			if err := c.netPlugin.TearDownPod(meta.NetNS, config.GetMetadata().GetNamespace(), name, id); err != nil {
+				glog.Errorf("failed to destroy network for container %s in sandbox %s: %v",
```

Comment:
`%q	a single-quoted character literal safely escaped with Go syntax.`.

For empty string, `%s` will print nothing ``, `%q` will print `""`.